libvcx/src/lib.rs

```diff
@@ -152,34 +152,41 @@ mod tests {
         ])
     }
 
-    fn send_cred_offer(did: &str, cred_def_handle: u32, connection: u32, credential_data: &str) -> u32 {
-        let credential_offer = issuer_credential::issuer_credential_create(cred_def_handle,
-                                                                           "1".to_string(),
-                                                                           did.to_string(),
-                                                                           "credential_name".to_string(),
-                                                                           credential_data.to_string(),
-                                                                           1).unwrap();
-        println!("sending credential offer");
-        issuer_credential::send_credential_offer(credential_offer, connection).unwrap();
+    fn create_and_send_cred_offer(did: &str, cred_def_handle: u32, connection: u32, credential_data: &str) -> u32 {
+        info!("create_and_send_cred_offer:: creating issuer credential");
```

Comment:
We use "create_and_send_cred_offer >>> creating issuer credential" convention (applies to other comments as well)